public/css/sequenceserver.css

```diff
@@ -583,6 +583,10 @@ input[type=checkbox] {
   cursor: pointer;
 }
 
+.sharing-panel a {
+  cursor: pointer;
+}
+
```

Comment:
I grepped for `cursor: pointer` in our `public/css` folder and came up with the following:

```
.cursor-pointer {
  cursor: pointer;
}
```

Thus an alternative is that you add the `cursor-pointer` CSS class to your `a` elements.

Reply:
For clarification, would that be something like: ` className="btn-link email-URL cursor-pointer"`?

Reply:
Yes